.github/workflows/pr_labels.yml

```diff
@@ -8,8 +8,6 @@ jobs:
   check_pr_labels:
     runs-on: ubuntu-18.04
     steps:
-      - name: Checkout
-        uses: actions/checkout@v2
```

Comment:
Unrelated change, but the labels workflow doesn't need to checkout the repo to evaluate labels.